src/js/fc.js

```diff
@@ -454,6 +454,8 @@ const FC = {
             dterm_lowpass2_hz:          0,
             dterm_lowpass2_type:        0,
             dyn_lpf_curve_expo:         0,
+            dyn_lpf_gyro_min_hz:        0,
```

Comment:
Where are these actually accessed?

Reply:
For changing dynamic notch to default values when toggling bidirectional dshot. I found them missing while verifying MSP commands in configuration and motor tab.

Reply:
@haslinghuis: Can you please point to the places in code where `dyn_lpf_gyro_min_hz` and `dyn_lpf_gyro_max_hz` are accessed?

Reply:
I got them from Firmware I guess, the dyn_lpf_gyro_min_hz and dyn_lpf_gyro_max_hz are indeed not being used. Should these two be added to the MSP.FILTER_CONFIG commands?